packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/lock-contract.test.ts

```diff
@@ -1,100 +1,115 @@
-import test, { Test } from "tape";
+import "jest-extended";
 import { v4 as uuidv4 } from "uuid";
+import Web3 from "web3";
+import { Account } from "web3-core";
+
 import { PluginRegistry } from "@hyperledger/cactus-core";
+import { PluginKeychainMemory } from "@hyperledger/cactus-plugin-keychain-memory";
+import { BesuTestLedger } from "@hyperledger/cactus-test-tooling";
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+import { PluginImportType } from "@hyperledger/cactus-core-api";
+
+import LockAssetContractJson from "../../../solidity/hello-world-contract/LockAsset.json";
+
 import {
   EthContractInvocationType,
   Web3SigningCredentialType,
   PluginLedgerConnectorBesu,
   PluginFactoryLedgerConnector,
-  //Web3SigningCredentialCactusKeychainRef,
   ReceiptType,
 } from "../../../../main/typescript/public-api";
-import { PluginKeychainMemory } from "@hyperledger/cactus-plugin-keychain-memory";
-import { BesuTestLedger } from "@hyperledger/cactus-test-tooling";
-import { LoggerProvider, LogLevelDesc } from "@hyperledger/cactus-common";
-import LockAssetContractJson from "../../../solidity/hello-world-contract/LockAsset.json";
-import Web3 from "web3";
-import { PluginImportType } from "@hyperledger/cactus-core-api";
-const level = "INFO";
-const label = "lock-contact test";
-const log = LoggerProvider.getOrCreate({ level, label });
-test("deploys contract via .json file", async (t: Test) => {
-  const logLevel: LogLevelDesc = "TRACE";
+
+const logLevel: LogLevelDesc = "TRACE";
```

Comment:
@petermetz Can you change this log level to something less verbose?

Reply:
@outSH: Yup, done!